### PR TITLE
Ship in a bottle

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Replaces the dragon boat in the chest with a ship in a bottle (which breaks to the dragon boat)

Thereis not really a difference but I thought it'd be cute

### CODE REVIEW

**Comment by jughu** [comment]
Dragonboat in a bottle????

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -384,6 +384,18 @@
 
 //Dragon Boat
 
+
+/obj/item/device/ship_in_a_bottle
```
Comment (RemieRichards) [inline code/modules/mining/lavaland/necropolis_chests.dm]: why is it ```/device```

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 2]
```diff
@@ -384,6 +384,18 @@
...
+	name = "ship in a bottle"
+	desc = "A tiny ship inside a bottle."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "ship_bottle"
+
+/obj/item/device/ship_in_a_bottle/attack_self(mob/user)
+	user << "You're not sure how they get the ships in these things, but you're pretty sure you know how to get it out."
+	playsound(user.loc, 'sound/effects/Glassbr1.ogg', 100, 1)
+	new /obj/vehicle/lavaboat/dragon(get_turf(src))
```
Comment (RemieRichards) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Why can I infinitely create boats.

Comment (KorPhaeron) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Part of the deep lore

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (xxalpha) [inline code/modules/mining/lavaland/necropolis_chests.dm]: This isn't a device Kor, it's a piece of art.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 3]
```diff
@@ -384,6 +384,18 @@
...
 
+
+/obj/item/device/ship_in_a_bottle
+	name = "ship in a bottle"
+	desc = "A tiny ship inside a bottle."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "ship_bottle"
+
+/obj/item/device/ship_in_a_bottle/attack_self(mob/user)
+	user << "You're not sure how they get the ships in these things, but you're pretty sure you know how to get it out."
```
Comment (xxalpha) [inline code/modules/mining/lavaland/necropolis_chests.dm]: Shouldn't this be an examine() message?

**Pull request merged**